### ISSUE TITLE
Manual build issue

### ISSUE THREAD

**Issue by Llompi** [rock88/moonlight-nx#58]
Could not get past manual build on http.cpp was getting error could not find #include <curl/curl.h> 
After troubleshooting found that switch-curl had to be added:

'''sudo dkp-pacman  --noconfirm -S devkitA64 switch-curl'''

Credit to @tlf30

**Comment by masagrator** [comment]
This is wrong. This repo doesn't use latest switch-curl, app won't connect to pc after building with latest curl. So by using this command you will generate broken app.

**Comment by rock88** [comment]
In the this [commit](https://github.com/rock88/moonlight-nx/commit/34a0ef83328155340f8acf0b36a241e1e7e3c7b6) I added precompiled libcurl of required version, so you need install libcurl with pacman just for a headers.

**Comment by Ganso** [comment]
I see lots of improvents in the main branch. If building is working, maybe it's time for a new release? ;-)
